server/auth/passport.js

```diff
@@ -0,0 +1,47 @@
+import localStrategy from "passport-local";
+const LocalStrategy = localStrategy.Strategy;
+
+import bcrypt from "bcrypt";
+
+export const applyPassportStrategy = passport => {
+  passport.serializeUser((user, done) => {
+    done(null, user.id);
+  });
+
+  passport.deserializeUser((id, done) => {
+    db.query(`SELECT * FROM User WHERE id like ${id}`, (err, rows) => {
+      console.log("go here");
+      console.log(rows[0]);
+      done(err, rows[0]);
+    });
+  });
+
+  passport.use(
+    "local",
+    new LocalStrategy(
+      {
+        usernameField: "email",
+        passwordField: "password"
+      },
+      (email, password, done) => {
+        console.log(email);
+        console.log(password);
+        db.query(
+          `SELECT id FROM User WHERE email like '${email}'`,
```

Comment:
what will this line result? Is it different from Select * ? try print it out using `console.log(rows[0]);`